src/erlcloud_as.erl

```diff
@@ -394,6 +395,55 @@ create_auto_scaling_group(Params, Config) ->
 
 
 %% --------------------------------------------------------------------
+%% @doc Update AutoScaling group.
+%% @end
+%% --------------------------------------------------------------------
+-spec update_auto_scaling_group(aws_autoscaling_group(), aws_config()) ->
+                                       ok | {error, term()}.
```

Comment:
The spec is missing a case for `update_auto_scaling_group(Params, Config)` call. Please change to something like `(aws_autoscaling_group() | list(), aws_config()) -> ...`